sci-libs/spyking-circus/spyking-circus-0.9.9.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6..8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Fast spike sorting by template matching"
+HOMEPAGE="https://github.com/spyking-circus/spyking-circus/"
+SRC_URI="https://github.com/spyking-circus/spyking-circus/archive/${PV}.tar.gz"
+LICENSE="CeCILL-2"
+
+SLOT="0"
+KEYWORDS="~amd64 ~amd64-linux ~x86 ~x86-linux"
```

Comment:
Please sort the keywords

Reply:
I didn't get it:
```bash
$ echo "~amd64 ~amd64-linux ~x86 ~x86-linux" | tr ' ' '\n' | sort | tr '\n' ' '
~amd64 ~amd64-linux ~x86 ~x86-linux
```
Aren't they already sorted?